src/core/core.tooltip.js

```diff
@@ -677,13 +677,15 @@ module.exports = function(Chart) {
 			};
 
 			// Before body lines
+			ctx.fillStyle = mergeOpacity(vm.bodyFontColor, opacity);
 			helpers.each(vm.beforeBody, fillLineOfText);
 
 			var drawColorBoxes = vm.displayColors;
 			xLinePadding = drawColorBoxes ? (bodyFontSize + 2) : 0;
 
 			// Draw body lines now
 			helpers.each(body, function(bodyItem, i) {
+				ctx.fillStyle = mergeOpacity(vm.labelTextColors[i], opacity);
```

Comment:
Could be:

```javascript
var textColor = mergeOpacity(vm.labelTextColors[i], opacity);
ctx.fillStyle = textColor;
```

then we can remove line 706 and avoid calling this `mergeOpacity` more than one time per body item, right?

Reply:
@Rittyan what do you think about the suggested change?

Reply:
@simonbrunel 
I think it is very good.
May I commit that change?(Include removing line 706)

Reply:
@Rittyan yup, you can push that change to this branch

Reply:
I fixed it. thanks for reviewing 🙏

Reply:
Thanks @Rittyan